auth_oauth_multi_token/models/res_users.py

```diff
@@ -24,60 +20,77 @@ def _generate_oauth_master_uuid(self):
         readonly=True,
         groups="base.group_system",
     )
+
     oauth_access_max_token = fields.Integer(
         string="Max Number of Simultaneous Connections", default=10, required=True
     )
-    oauth_master_uuid = fields.Char(
+
+    # use the oauth_access_token field as oauth_master_uuid
+    oauth_access_token = fields.Char(
         string="Master UUID",
-        copy=False,
```

Comment:
Why are you removing these attributes?

Reply:
You are right. I thought False is the default, but it is not. @kobros-tech could you please readd the copy=False?